hub/util/remove_cache.py

```diff
@@ -1,8 +1,9 @@
+from email.mime import base
```

Comment:
Why do you need email package here? and I don't see `base` code used below.

Reply:
seems like accidental auto import, removing